src/login/logind.c

```diff
@@ -24,6 +24,7 @@
 #include "selinux-util.h"
 #include "signal-util.h"
 #include "strv.h"
+#include "terminal-util.h"
```

Comment:
nitpick: in the commit msg, can you prefix the issue id with `#` so that it becomes clickable in the github ui?

Reply:
sure.